src/herbivoresAndCarnivores.js

```diff
@@ -2,14 +2,43 @@
 
 class Animal {
   // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    Animal.addAnimal(this);
+  }
+
+  static addAnimal(animal) {
+    this.alive.push(animal);
+  }
 }
 
+Animal.alive = [];
+
 class Herbivore extends Animal {
   // write your code here
+  constructor() {
+    super();
+    this.hidden = false;
+  }
+
+  hide() {
+    this.hidden = this.hidden ? 0 : true;
```

Comment:
[checklist](https://github.com/mate-academy/js_herbivores_and_carnivores/blob/master/checklist.md) 3rd item, do not toggle hidden value, just set it to true